docs/ref/models/instances.txt

```diff
@@ -360,6 +360,10 @@ method. See :ref:`overriding-model-methods` for more details.
 
 The model save process also has some subtleties; see the sections below.
 
+For details on using the ``force_insert`` and ``force_update`` arguments, see
+:ref:`ref-models-force-insert`. Details about the ``update_fields`` argument
+can be found in the :ref:`ref-models-update-fields` section.
```

Comment:
Maybe this is too minor to mention, but I think this would be better as the first paragraph, IMO. For one, it is right after mention of the arguments. And it seems like more basic / important info than customizing (power users) and subtleties.

Reply:
Changed.